src/lib/cache/subCache.cpp

```diff
@@ -666,21 +666,24 @@ void subCacheItemInsert(CachedSubscription* cSubP)
 */
 void subCacheItemInsert
 (
-  const char*               tenant,
-  const char*               servicePath,
-  SubscribeContextRequest*  scrP,
-  const char*               subscriptionId,
-  int64_t                   expirationTime,
-  int64_t                   throttling,
-  RenderFormat              renderFormat,
-  bool                      notificationDone,
-  int64_t                   lastNotificationTime,
-  StringFilter*             stringFilterP,
-  const std::string&        status,
-  const std::string&        q,
-  const std::string&        geometry,
-  const std::string&        coords,
-  const std::string&        georel
+  const char*                   tenant,
+  const char*                   servicePath,
+  const std::string&            url,
+  const EntityIdVector&         entityIdVector,
+  const AttributeList&          attributeList,
+  const NotifyConditionVector&  notifyConditionVector,
+  const char*                   subscriptionId,
```

Comment:
Indentation + changed 

    SubscribeContextRequest*  scrP

for

    const std::string&            url,
    const EntityIdVector&         entityIdVector,
    const AttributeList&          attributeList,
    const NotifyConditionVector&  notifyConditionVector,